content/tutorials/get-started/set-up-secure-signing.md

```diff
@@ -0,0 +1,134 @@
+
+    In the `rippled`'s config file, modify the `[port_rpc_admin_local]` and `[port_ws_admin_local]` stanzas to use your server's private-LAN address:
+
+        [port_rpc_admin_local]
+      	port = 5005
+      	ip = 10.1.0.0 # Changed from 127.0.0.1
+      	admin = 10.1.1.2 # Only the client with this IP is treated as admin
+      	protocol = http
+
+        [port_ws_admin_local]
+      	port = 6006
+      	ip = 10.1.0.0 # Changed from 127.0.0.1
+      	admin = 10.1.1.2 # Only the client with this IP is treated as admin
+      	protocol = ws
+
+4. If your `rippled` machine runs a software firewall, configure the firewall to accept connections on your local private network at the JSON-RPC and Websocket ports you configured in the previous step. (Ports `5005` and `6006` in the previous example.)
+
+    The exact configuration depends on your firewall software.
+
+5. On the machine(s) that will submit transactions, connect to your server using your `rippled` server's private IP address and the configured ports. Use the [sign method][] (for single signatures) or [sign_for method][] (for multi-signatures).
+
+    The example configuration from the previous steps uses port `5005` for JSON-RPC connections and port `6006` for Websocket connections.
+
+6. Keep the `rippled` server running, updated, and in sync with the network while you're using it.
+
+    **Note:** You _can_ turn off your `rippled` server when you're not sending transactions, but it can take up to 15 minutes to sync with the network when you start it up again.
+
+
+## Use a Client Library with Local Signing
+
+[![Diagram of using a client library with local signing](img/secure-signing-client-library.png)](img/secure-signing-client-library.png)
+
+This configuration uses a client library in the programming language you are using to sign transactions locally. You need a client library for the programming language you use. Ripple publishes the following client libraries that can sign transactions for the XRP Ledger locally:
+
+- **RippleAPI (ripple-lib) for JavaScript**
+    - [Setup](get-started-with-rippleapi-for-javascript.html)
+    - [API Reference](rippleapi-reference.html)
+- **ripple-libpp for C++**
+    - [Source and Documentation](https://github.com/ripple/ripple-libpp/)
+
+If you use a client library not published by Ripple, make sure it uses proper, secure implementations of the signing algorithm(s) it implements. (For example, if it uses the default ECDSA algorithm, it should also use deterministic nonces as described in [RFC6979](https://tools.ietf.org/html/rfc6979).) All of Ripple's published libraries listed above follow industry best practices.
+
+For best security, be sure to keep your client library updated to the latest stable version.
+
+### Example Local Signing with RippleAPI
+
+The following code sample shows how to sign transaction instructions locally with RippleAPI for JavaScript:
+
+```js
+{% include '_code-samples/secure-signing/js/signPayment.js' %}
```

Comment:
We need to highlight that this snippet is _insecure_ and should not be used as is, in that it includes the key directly in the script file.

Reply:
Let's try to figure out a plausible way of demonstrating securely importing the secret instead